### DISCUSSION TITLE
Trouble installing on Ubuntu 20.04

### DISCUSSION THREAD

**Discussion by jacksonsystems** [Budibase/budibase#6380] [Help]
Hi all

I am trying to install on an Ubuntu 20.04 Virtual machine - upon running 'npm i -g @budibase/cli'

I get the following error:

npm ERR! code EACCES
npm ERR! syscall rename
npm ERR! path /usr/local/bin/budi
npm ERR! dest /usr/local/bin/.budi-LlJfBV7U
npm ERR! errno -13
npm ERR! Error: EACCES: permission denied, rename '/usr/local/bin/budi' -> '/usr/local/bin/.budi-LlJfBV7U'
npm ERR!  [Error: EACCES: permission denied, rename '/usr/local/bin/budi' -> '/usr/local/bin/.budi-LlJfBV7U'] {
npm ERR!   errno: -13,
npm ERR!   code: 'EACCES',
npm ERR!   syscall: 'rename',
npm ERR!   path: '/usr/local/bin/budi',
npm ERR!   dest: '/usr/local/bin/.budi-LlJfBV7U'
npm ERR! }
npm ERR!
npm ERR! The operation was rejected by your operating system.
npm ERR! It is likely you do not have the permissions to access this file as the current user
npm ERR!
npm ERR! If you believe this might be a permissions issue, please double-check the
npm ERR! permissions of the file and its containing directories, or try running
npm ERR! the command again as root/Administrator.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/sa/.npm/_logs/2022-06-18T21_28_14_714Z-debug-0.log

I am not sure what I need to do from here.

I have ytried running as sudo alos with the same error.

Thanks

**Comment by shogunpurple** [comment]
@jacksonsystems seems like an NPM permissions issue on `/usr/local/bin`. Try:

`sudo chown -R $USER /usr/local/bin`
